docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-importing-tokens.mdx

```diff
@@ -0,0 +1,67 @@
+---
+keywords: [beginner, governance, tutorial, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import '/src/components/CenterImages/center.scss';
+
+# Import ICRC tokens
+
+<MarkdownChipRow labels={["Beginner", "Governance", "Tutorial"]} />
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zcXmarBk2L4?si=OIxq9LwY0DLzHvE1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## Overview
+
+The NNS dapp focuses on providing a simple user interface where community members can interact with different on-chain governance systems deployed on the Internet Computer.
+
+To extend wallet capabilities, users of the NNS dapp can now import any token that supports ICRC-1 no longer limited to ICP and SNS governance tokens. This includes all ICRC-2 and ICRC-3 tokens.
+
+## How to find canister IDs
+
+There are several ways to find ledger canister IDs of ICRC-1 tokens. The tutorial video above mentions two. This is a more comprehensive list.
```

Comment:
I would lead the paragraph with why you even talk about canister IDs: 

```suggestion
To import a token to you NNS dapp wallet so that you can.... , you need to specify the token by the canister ID of the token's ledger canister. [Optionally also index: why do people need this? ]

There are several ways to find ledger canister IDs of ICRC-1 tokens. The tutorial video above mentions two. This is a more comprehensive list.
```

Reply:
(this could also be the end of the overview section and be a bridge to the "how to find canister IDs")